src/module/actor/modifiers.ts

```diff
@@ -642,8 +642,8 @@ class DiceModifierPF2e implements BaseRawModifier {
             : null;
 
         this.predicate = new PredicatePF2e(params.predicate ?? []);
-        this.enabled = this.predicate.test([]);
-        this.ignored = !this.enabled;
+        this.enabled = params.enabled ?? this.predicate.test([]);
+        this.ignored = params.ignored ?? !this.enabled;
```

Comment:
How about we just remove the empty predicate test?

Reply:
we could replace it for testing if the predicate is empty instead. I think the intent is to say "if its not tested and it needs to be, default to disabled". Might break something if we don't have anything at all here.

Reply:
They get tested in @system/damage/weapon, so it's covered there.